src/custom/constants/index.ts

```diff
@@ -57,7 +57,7 @@ export const V_COW_CONTRACT_ADDRESS: Partial<Record<number, string>> = {
   // [ChainId.MAINNET]: GPv2Settlement[ChainId.MAINNET].address,
   // [ChainId.RINKEBY]: GPv2Settlement[ChainId.RINKEBY].address,
   // [ChainId.XDAI]: GPv2Settlement[ChainId.XDAI].address,
-  [ChainId.RINKEBY]: '0x8f07778b703f9feb93521fc5230e859dbd032b6b',
+  [ChainId.RINKEBY]: '0xac17Ae2BcAF9Ed181C4Ab10B179548d6aa30C92E',
```

Comment:
This is why it didn't work before, I was using the wrong contract address...

Reply:
https://media4.giphy.com/media/xT9DPJVjlYHwWsZRxm/giphy.gif?cid=790b76118dea2e68410684f9b11b5c4ac93fa1130bf1af15&rid=giphy.gif&ct=g